### PR TITLE
Added style to images

### PR DESCRIPTION
**Pull request by Jomalagon**
style images was changed, shadows and animation were added.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -112,6 +116,12 @@ button {
     height: 300px;
     width: 300px;
     margin: 35px;
+    box-shadow: 10px 10px 4px rgba(101, 110, 78, 0.616)
+}
+.container__img:hover {
```
Comment (axelzo) [inline style.css]: Try to follow the same pattern, i think it would be better `.container__image:hover` like classes above.

**Review by axelzo — CHANGES_REQUESTED** [review]

**Comment by axelzo** [comment]
@Jomalagon Thanks for this, it looks awesome. 👏  I left a minor comment. let me know if you have doubts.

File: style.css (hunk 1 above)
Comment (Jomalagon) [reply style.css]: it's ready

**Comment by Jomalagon** [comment]
@axelzo
css was made uniform with BEM, shadow and animation were added to the images.

File: style.css (hunk 1 above)
Comment (Jomalagon) [reply style.css]: more changes were added, I hope you like them.

**Comment by Jomalagon** [comment]
@axelzo A logo was added, some labels were changed in the css and also animation was given to the images.

**Comment by Jomalagon** [comment]
@axelzo  Added BEM nomenclature in CSS and 
unnecessary divs removed.

**Comment by axelzo** [comment]
@Jomalagon looks great!. please squahs it.

**Review by axelzo — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (Jomalagon) [reply style.css]: it's ready

**Comment by Jomalagon** [comment]
@axelzo a new branch was created so that it did not conflict to squash commits.

**Pull request merged**